peersession_test.go

```diff
@@ -16,6 +16,7 @@ import (
 
 const (
 	TEST_NAME     = "bootstrap-unittest"
+	KEY_DIFF      = 10
```

Comment:
Just put `0`, it's not important in this test and it shouldn't slow it down

Reply:
Done